lib/app/settings/presentation/settings_view.dart

```diff
@@ -30,25 +30,29 @@ class SettingsView extends ConsumerWidget {
                 child: ListView(
                   padding: const EdgeInsets.symmetric(horizontal: 16),
                   children: [
-                    SettingToggleCard(
-                      title: 'Automatically Start Focus',
-                      enabled: state.automaticallyStartFocus,
-                      onChanged: notifier.updateAutomaticallyStartFocus,
-                    ),
-                    const SizedBox(height: 8),
                     SettingDurationCard(
                       duration: state.focusDuration,
                       icon: '🧘‍♂️',
                       title: 'Focus Duration',
                       onDurationChanged: notifier.updateFocusDuration,
                     ),
                     const SizedBox(height: 8),
+                    const _SettingsDivider(),
```

Comment:
@whopavan I think focus duration and focus autostart toggle should be together. And break duration and break auto start toggle should be together.

Reply:
Ok. Does this look fine?
<img width="580" alt="image" src="https://github.com/user-attachments/assets/7b8afb0a-3e03-40b4-aa4b-ebf316eb72a2">

Reply:
Yup